solution_1.rb

```diff
@@ -0,0 +1,91 @@
+class Tutorial
+
+  def solution(x)
+    str = x.to_s
+    num = str.chars.sort.join.reverse!
```

Comment:
For the last line of function, it's auto return value, so
```suggestion
    str.chars.sort.join.reverse!
```